go/pkg/cli/show.go

```diff
@@ -161,7 +163,32 @@ func showExperiment(au aurora.Aurora, out io.Writer, proj *project.Project, exp
 	return nil
 }
 
-func writeExperimentCommon(au aurora.Aurora, w *tabwriter.Writer, exp *project.Experiment, experimentRunning bool) {
+func isInterestingPythonPackage(pkg string) bool {
+	switch pkg {
+	case "numpy":
+		return true
+	case "pandas":
+		return true
+	case "sklearn":
+		return true
+	case "tensorflow":
+		return true
+	case "torch":
+		return true
+	}
+	return false
+}
+
+func filter(ss []string, test func(string) bool) (ret []string) {
```

Comment:
maybe move this to `slices.FilterStrings`?

Reply:
good idea